CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
+
```

Comment:
I quite like the additional context/paragraph you included in the CLI branch:
```suggestion

We're happy to receive feature requests and PRs. If your change is nontrivial,
please open an [issue](https://github.com/fastly/cli/issues/new) to discuss the
idea and implementation strategy before submitting a PR.

```

Reply:
Fixed link to `go-fastly`.